docs/data-sources/flex_cluster.md

```diff
@@ -0,0 +1,90 @@
+# Data Source: mongodbatlas_flex_cluster
+
+`mongodbatlas_flex_cluster` describes a flex cluster.
+
+-> **NOTE**: Flex Clusters are currently in Preview. To use this feature, you must take the following actions:
+1. Enable the `Atlas Uss Enabled` Preview Feature in your project (contact [MongoDB Support](https://www.mongodb.com/services/support)).
```

Comment:
can we double check here that Uss will still be the name?

Reply:
I'm actually unsure if users will need to manually enable the feature at all when Flex is released. Will add this question to the agenda for the Flex Sync.

Reply:
Given the conversation in today's flex sync, I'll remove these action items.